libs/wingsdk/src/target-sim/service.inflight.ts

```diff
@@ -5,23 +5,23 @@ import {
   IServiceStopHandlerClient,
   SERVICE_FQN,
 } from "../cloud";
-import { Sandbox } from "../shared/sandbox";
+import { LegacySandbox } from "../shared/sandbox";
 import { ISimulatorContext, ISimulatorResourceInstance } from "../simulator";
 import { TraceType } from "../std";
 
 export class Service implements IServiceClient, ISimulatorResourceInstance {
   private readonly context: ISimulatorContext;
   private readonly entrypoint: string;
   private readonly autoStart: boolean;
-  private readonly sandbox: Sandbox;
+  private readonly sandbox: LegacySandbox;
   private running: boolean = false;
   private onStop?: IServiceStopHandlerClient;
 
   constructor(props: ServiceSchema["props"], context: ISimulatorContext) {
     this.context = context;
     this.entrypoint = resolve(context.simdir, props.sourceCodeFile);
     this.autoStart = props.autoStart;
-    this.sandbox = new Sandbox(this.entrypoint, {
+    this.sandbox = new LegacySandbox(this.entrypoint, {
```

Comment:
Open an issue about this (cloud.Service can't use the updated Sandbox because the API tries sending a function closure across HTTP)

Reply:
Any thoughts on how to address this?

Reply:
Probably https://github.com/winglang/wing/issues/4781